src/app/src/App.css

```diff
@@ -1,16 +1,19 @@
 .edit-poly-marker {
     margin-left: -4px !important;
     margin-top: -4px !important;
-    width: 8px !important;
-    height: 8px !important;
+    width: 10px !important;
+    height: 10px !important;
```

Comment:
These are defined as `8px` and `6px` in Figma https://www.figma.com/file/tJPZ0lkT5C3SAyS2g2R1aA/IoW---Boundary-Sync-Tool?node-id=498%3A14829, but we use larger sizes here because the box sizing model is different.

With the `border-box` sizing model, borders grow inwards. I tried using `content-box` but that messed up the layout. Going with slightly larger sizes seemed like a good approximation.